src/ctapipe/instrument/optics.py

```diff
@@ -263,3 +270,141 @@ def __repr__(self):
 
     def __str__(self):
         return self.name
+
+
+class PSFModel(TelescopeComponent):
+    def __init__(self, subarray, **kwargs):
+        """
+        Base component to describe image distortion due to the optics of the different cameras.
+        """
+        super().__init__(subarray=subarray, **kwargs)
+
+    @abstractmethod
+    def pdf(self, *args):
```

Comment:
This needs a docstring so that subclass implementers know what is expected.

Also the signature should be the same for all classes, so it should be explicit here in the interface definition.